packages/constellation/src/components/ThemeProvider/ThemeProvider.tsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable import/no-unresolved */
+import '../styles.css'
+
+import React, { useEffect, useState } from 'react'
+
+const ThemeProvider = ({
```

Comment:
Could you define the ThemeProvider type in the `ThemeProvider.types.ts` file
as `type ThemeProviderComponent = FC<ThemeProviderProps>`